contracts/zero-ex/contracts/src/transformers/FillQuoteTransformer.sol

```diff
@@ -149,15 +160,19 @@ contract FillQuoteTransformer is
     /// @dev The exchange proxy contract.
     INativeOrdersFeature public immutable zeroEx;
 
+    // @dev the otc orders feature
+    IOtcOrdersFeature public immutable otcOrders;
+
```

Comment:
we want to call `fillOtcOrder` on the `zeroEx` address, so you should just change the interface of the above `zeroEx` immutable variable to `IZeroEx` instead of introducing another variable here